bloom-housing/bloom
# fix: may the force be with you

This PR addresses https://github.com/metrotranscom/doorway/issues/868


## Description
Tears down the db with force during the teardown process

## How Can This Be Tested/Reviewed?
run `yarn db setup` and it should successfully tear down and rebuild your db
once this is in doorway we'll more easily be able to test it under dev conditions

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
